src/Function.cpp

```diff
@@ -562,6 +570,21 @@ Function::OutputForwardDecl(std::ostream &out)
 	OutputHeader(out);
 	if(func_attr_inline)
 		out << " __attribute__((always_inline))";
+	if(func_attr_aligned){
+		int probability__BIGGEST_ALIGNMENT__ = 58;
+		int value = 1;
+		bool use__BIGGEST_ALIGNMENT__ = rnd_flipcoin(probability__BIGGEST_ALIGNMENT__);
+		if (use__BIGGEST_ALIGNMENT__)
+			value = __BIGGEST_ALIGNMENT__;
+		else{
+			int power = rnd_upto(__BIGGEST_ALIGNMENT__);
```

Comment:
Limiting power to 16 instead of __BIGGEST_ALIGNMENT__
Is this you were suggesting?

Any other changes?

Reply:
> Limiting power to 16 instead of **BIGGEST_ALIGNMENT**
> Is this you were suggesting?

Yes.

> 
> Any other changes?

I'll test the patch after this fix.